components/webfield/ProgramChairConsole/PaperStatus.js

```diff
@@ -143,13 +276,25 @@ const PaperStatus = ({ pcConsoleData, loadReviewMetaReviewData }) => {
     } else {
       const { notes, noteNumberReviewMetaReviewMap } = pcConsoleData
       if (!notes) return
-      const tableRows = [...(noteNumberReviewMetaReviewMap.values() ?? [])]
+      const actualNotes = noteContentField
+        ? notes.filter((note) => Object.keys(note.content).includes(noteContentField.field))
```

Comment:
In JavaScript to check if an object has a certain field you can just do `note.content[noteContentField.field]`. Since note content fields are always objects, if this returns false you know the note doesn't have that field.